src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -452,10 +452,28 @@ namespace AppInstaller::CLI::Workflow
             if (symlinkPath.has_value())
             {
                 const std::filesystem::path& symlinkPathValue = symlinkPath.value().GetValue<Value::Type::UTF16String>();
-
-                if (!std::filesystem::remove(symlinkPathValue))
+                if (std::filesystem::is_symlink(std::filesystem::symlink_status(symlinkPathValue)))
                 {
-                    AICLI_LOG(CLI, Info, << "Portable symlink not found; Unable to delete portable symlink: " << symlinkPathValue);
+                    const auto& targetPath = uninstallEntry[PortableValueName::PortableTargetFullPath];
+                    if (targetPath.has_value())
+                    {
+                        const std::filesystem::path& symlinkTargetPath = std::filesystem::read_symlink(symlinkPathValue);
+                        const std::filesystem::path& targetPathValue = targetPath.value().GetValue<Value::Type::UTF16String>();
+                        if (symlinkTargetPath != targetPathValue)
+                        {
+                            AICLI_LOG(CLI, Info, << "Portable symlink not deleted; Symlink points to a different target exe: " << symlinkTargetPath <<

```

Comment:
Since the context reporter is reporting a warning, I would expect the log to be a warning also
```suggestion
                            AICLI_LOG(CLI, Warning, << "Portable symlink not deleted; Symlink points to a different target exe: " << symlinkTargetPath <<
```